### PR TITLE
Feature database schema

### LINKED ISSUE #2: Back-end: Fill out db.js with database schema

### LINKED ISSUE #4: Fill out db/model.js with sequelize methods for populating dashboard

### PR DESCRIPTION
**Pull request by alanzfu**
#### What's this PR do?
- fills out config file with db connection
- uses pg-promise to create getAllClients method to interact with database
#### What are the important parts of the code?

config.js: creates connection
schema.js: creates schemas for salesperson/client/salesperson_client
index.js: exports modules
psql folder: holds all query strings in variables within their own js files
#### How should this be tested by the reviewer?
- if tables are created in a 'recon' database
- if getAllClients returns an array of client objects
#### Is any other information necessary to understand this?
- to get postgres up and running, install postgresapp and follow instructructions. then create 'recon' database
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

closes #2 
closes #4

### CODE REVIEW

File: db/psql/getAllClients.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var db = require('../config.js');
+//This function gets all clients from salesperson_client for a specific salesperson
+
+
+var getAllClientsString = "SELECT * FROM client INNER JOIN salesperson_client on (salesperson_client.client_id = client.client_id) WHERE salesperson_client.salesperson_id = $1"
+
+module.exports = function(salesPersonId){
```
Comment (rebeccaryang) [inline db/psql/getAllClients.js]: Can we add a next callback as a second argument to the exported function, that gets called after the database call and passes in (error, result) as the arguments? (null, result) for the not error case and (error, null) for the error case

Comment (alanzfu) [inline db/psql/getAllClients.js]: done! it's promisifed so you only get the results. so invocation is callback(result)

File: db/psql/getAllClients.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+var getAllClientsString = "SELECT * FROM client INNER JOIN salesperson_client on (salesperson_client.client_id = client.client_id) WHERE salesperson_client.salesperson_id = $1"
+
+module.exports = function(salesPersonId, callback){
+    return db.query(getAllClientsString, salesPersonId)
+    .then(function(clients){
+      callback(clients);
+      return clients;
+    })
+    .catch(function(error){
+      return error;
```
Comment (rebeccaryang) [inline db/psql/getAllClients.js:14]: callback(error,null) ??

File: db/psql/getAllClients.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+var db = require('../config.js');
+//This function gets all clients from salesperson_client for a specific salesperson
+
+
+var getAllClientsString = "SELECT * FROM client INNER JOIN salesperson_client on (salesperson_client.client_id = client.client_id) WHERE salesperson_client.salesperson_id = $1"
+
+module.exports = function(salesPersonId, callback){
+    return db.query(getAllClientsString, salesPersonId)
+    .then(function(clients){
+      callback(clients);
```
Comment (rebeccaryang) [inline db/psql/getAllClients.js:10]: callback(null, clients)

**Pull request merged**